gatsby-site/src/components/BackgroundWave/BackgroundWave.css

```diff
@@ -0,0 +1,15 @@
+.wave-container {
+    position: absolute;
+}
+
+.purple-wave {
+    top: -550px;
+    z-index: -2 !important;
+    position: absolute;
```

Comment:
purple and blue wave will inherit absolute from their parent, wave-container.  No need to type it more than once.